test/__snapshots__/serialize.test.ts.snap

```diff
@@ -12380,6 +12380,1890 @@ function __f0() {
 "
 `;
 
+exports[`serialize event bridge aws put events 1`] = `
```

Comment:
Looks significantly less than 50k, but i am totally eye balling.

Reply:
yeah, I didn't look, just didn't fail the test (50k).

Reply:
Time to ratchet that down some more?